liquidator/abi/IERC20.json

```diff
@@ -0,0 +1,563 @@
+[
```

Comment:
do we need to import this ABI? we have ERC20s elsewhere in the repo that can be re-used, right?

Reply:
Once `truffle compile` has been run you should be able to get this from `/core/build/contracts`

Reply:
Otherwise you can access the artefact directly from truffle. see [here](https://github.com/UMAprotocol/protocol/blob/9120d52051473e630f2cb1e626c5385ca39e3283/liquidator/index.js#L21)